be/src/util/mysql_row_buffer.cpp

```diff
@@ -26,6 +26,8 @@
 
 namespace doris {
 
+#define NOT_DYNAMIC_MODE  (_dynamic_mode == 0)           
```

Comment:
Can you give more comments for this option to help others?

Reply:
OK. Will add comment in .h